rfc-175-frontend-fewer-apps.md

```diff
@@ -0,0 +1,254 @@
+---
+status: proposed
+implementation: proposed
+status_last_reviewed:
+---
+
+# Consolidate Frontend Content-Rendering Apps
+
+## Summary
+
+This is a refinement of the suggestions in [RFC-174] about reducing the number of frontend rendering applications, reducing its scope to the applications that deal exclusively with rendering content items. By dealing only with the four applications that exclusively render content items, we improve standardisation and reduce maintenance costs, and the work is worthwhile even if we can only achieve sections of it. Because a large part of the problem and a lot of the solution remains the same, we have divided this RFC into two parts - an initial section that describes how we differ from [RFC-174], and a more verbose section detailing the approach. If you have already read or commented on [RFC-174], you can read the [How this proposal differs](#how-this-proposal-differs) section and skip the rest of the RFC. If you are new to this proposal, feel free to skip to the [Full Exploration of the Problem](#full-exploration) section.
```

Comment:
Is it worth being explicit that RFC-174 was closed and not accepted? The language in this RFC made me assume 174 had been merged and then descoped here in 175.

Reply:
Ah yes, good point - and we should properly merge 174 in marked as rejected(? or whatever we do with rejected RFCs)

Reply:
Done.